pkg/operation/botanist/component/istio/ingress_gateway.go

```diff
@@ -111,3 +121,280 @@ func addSuffixToManifestsName(charts *chartrenderer.RenderedChart, suffix string
+		{
+			name: "to-shoot-vpn-seed-server",
+			transform: func(obj *networkingv1.NetworkPolicy) func() error {
+				return func() error {
+					obj.Annotations = map[string]string{
+						v1beta1constants.GardenerDescription: fmt.Sprintf("Allows Egress from pods labeled with "+
+							"'%s=%s' to shoot vpn servers with label '%s=%s'.", v1beta1constants.LabelApp, v1beta1constants.DefaultIngressGatewayAppLabelValue,
+							v1beta1constants.LabelApp, v1beta1constants.DeploymentNameVPNSeedServer),
+					}
+					obj.Spec = networkingv1.NetworkPolicySpec{
+						PodSelector: metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								v1beta1constants.LabelApp: v1beta1constants.DefaultIngressGatewayAppLabelValue,
+							},
+						},
+						PolicyTypes: []networkingv1.PolicyType{networkingv1.PolicyTypeEgress},
+						Egress: []networkingv1.NetworkPolicyEgressRule{{
+							To: []networkingv1.NetworkPolicyPeer{{
+								PodSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										v1beta1constants.LabelApp:   v1beta1constants.DeploymentNameVPNSeedServer,
+										v1beta1constants.GardenRole: v1beta1constants.GardenRoleControlPlane,
+									},
+								},
+								NamespaceSelector: &metav1.LabelSelector{},
+							}},
+							Ports: []networkingv1.NetworkPolicyPort{{
+								Protocol: protocolPtr(corev1.ProtocolTCP),
+								Port:     intStrPtr(1194),
+							}},
+						}},
+					}
+					return nil
+				}
+			},
+		},
+		{
+			name: "to-reversed-vpn-auth-server",
+			transform: func(obj *networkingv1.NetworkPolicy) func() error {
+				return func() error {
+					obj.Annotations = map[string]string{
+						v1beta1constants.GardenerDescription: fmt.Sprintf("Allows Egress from pods labeled with "+
+							"'%s=%s' to reversed vpn auth servers with label '%s=%s' in namespace %s.",
+							v1beta1constants.LabelApp, v1beta1constants.DefaultIngressGatewayAppLabelValue,
+							v1beta1constants.LabelApp, "reversed-vpn-auth-server",
+							v1beta1constants.GardenNamespace,
+						),
+					}
+					obj.Spec = networkingv1.NetworkPolicySpec{
+						PodSelector: metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								v1beta1constants.LabelApp: v1beta1constants.DefaultIngressGatewayAppLabelValue,
+							},
+						},
+						PolicyTypes: []networkingv1.PolicyType{networkingv1.PolicyTypeEgress},
+						Egress: []networkingv1.NetworkPolicyEgressRule{{
+							To: []networkingv1.NetworkPolicyPeer{{
+								PodSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										v1beta1constants.LabelApp: "reversed-vpn-auth-server",
+									},
+								},
+								NamespaceSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										v1beta1constants.LabelRole: v1beta1constants.GardenNamespace,
+									},
+								},
+							}},
+							Ports: []networkingv1.NetworkPolicyPort{{
+								Protocol: protocolPtr(corev1.ProtocolTCP),
+								Port:     intStrPtr(9001),
```

Comment:
Please export the port constant from `external_authz_server.go` and reuse it here.